heron/healthmgr/src/java/com/twitter/heron/healthmgr/sensors/MetricsCacheMetricsProvider.java

```diff
@@ -0,0 +1,161 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package com.twitter.heron.healthmgr.sensors;
+
+import java.net.HttpURLConnection;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.microsoft.dhalion.api.MetricsProvider;
+import com.microsoft.dhalion.metrics.ComponentMetrics;
+import com.microsoft.dhalion.metrics.InstanceMetrics;
+
+import com.twitter.heron.proto.system.Common.StatusCode;
+import com.twitter.heron.proto.tmaster.TopologyMaster;
+import com.twitter.heron.proto.tmaster.TopologyMaster.MetricInterval;
+import com.twitter.heron.proto.tmaster.TopologyMaster.MetricResponse.IndividualMetric;
+import com.twitter.heron.proto.tmaster.TopologyMaster.MetricResponse.IndividualMetric.IntervalValue;
+import com.twitter.heron.proto.tmaster.TopologyMaster.MetricResponse.TaskMetric;
+import com.twitter.heron.spi.utils.NetworkUtils;
+
+import static com.twitter.heron.healthmgr.HealthManager.CONF_METRICS_SOURCE_URL;
+import static com.twitter.heron.metricscachemgr.MetricsCacheManagerHttpServer.PATH_STATS;
```

Comment:
It seems this provider depends on `metricscachemgr-java` in BUILD because of this import. Would it be better to rather define the constant here?